Sources/NIOTransportServices/NIOTSConnectionBootstrap.swift

```diff
@@ -261,6 +274,528 @@ public final class NIOTSConnectionBootstrap {
     }
 }
 
+// MARK: Async connect methods with arbitrary payload
+
+@available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
+extension NIOTSConnectionBootstrap {
+    /// Specify the `host` and `port` to connect to for the TCP `Channel` that will be established.
+    ///
+    /// - Parameters:
+    ///   - host: The host to connect to.
+    ///   - port: The port to connect to.
+    ///   - channelInitializer: A closure to initialize the channel. The return value of this closure is returned from the `connect`
+    ///   method.
```

Comment:
```suggestion
    ///   - channelInitializer: A closure to initialize the channel. The return value of this closure is returned from the `connect`
    ///     method.
```

Reply:
Not sure about this. This is the indention that Xcode suggests me

Reply:
I think Xcode just indents to where the next bullet point should be.

Moreover, I don't think that's a good reason: docs are for humans so we should make it easy for humans to read them.